Lib/fontTools/feaLib/ast.py

```diff
@@ -1146,7 +1146,9 @@ def build(self, builder):
     def asFea(self, indent=""):
         res = "pos base {}".format(self.base.asFea())
         for a, m in self.marks:
-            res += " {} mark @{}".format(a.asFea(), m.name)
+            res += ("\n" + indent + SHIFT + "{} mark @{}".format(
```

Comment:
indent this as black would?
not sure if the rest of the module was ever blackened; if it was, then keep it black; otherwise we'll do that some other time.